cdap-security-spi/src/test/java/co/cask/cdap/security/spi/authorization/AuthorizerTest.java

```diff
@@ -46,6 +49,10 @@ public void testSimple() throws UnauthorizedException {
     authorizer.grant(namespace, user, Collections.singleton(Action.READ));
     authorizer.enforce(namespace, user, Action.READ);
 
+    Set<Privilege> expectedPrivileges = new HashSet<>();
```

Comment:
we can suppress the tests in DatasetBasedAuthorizer. But we should add tests in AuthorizerTest. Otherwise its not complete. AuthorizerTest is for use not just by DatasetBasedAuthorizer but by all authorizers.